src/CarousleCardsRow.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import cardPropTypes from './modelPropTypes'
+import Card from './Card'
+import Slide from 'react-slick'
+
+const CarousleCardsRow = (props) => {
+  const { CardClass, cards, slideSettings } = props
+  const { className, cardContainerClassName, speciesIconHeight, imageIconHeight, hoverColour } = props
+  slideSettings.slidesToShow = slideSettings.slidesToShow >= cards.length ? cards.length : slideSettings.slidesToShow
+
+  const CardContainer = styled.div`
+    /* border-radius: 8px; */
+    :hover {
+      background: ${hoverColour};
+    }
+  `
+  const cardsDisplay = Array.isArray(cards) && cards.map((card, index) =>
+    <CardContainer className={cardContainerClassName} key={index}>
+      <CardClass
+        {...card}
+        imageIconHeight={imageIconHeight}
+        speciesIconHeight={speciesIconHeight}/>
+    </CardContainer>)
+
+  return (
+    <div className={className}>
+      {
+        <Slide id={`slide`} 
```

Comment:
As a general rule don’t use `id` in your components. They’re source of all sorts of trouble (e.g. you render two components on the same page).